devo/sender/data.py

```diff
@@ -85,10 +87,15 @@ def __init__(self, message: str):
         :param message: Message describing the exception. It will be also
          used as `args` attribute in `Exception` class
         """
+        if isinstance(message, str) is False:
```

Comment:
For clarity, is this better?

 ```if not isinstance(message, str)``` 

I'm wondering if the overridden __str__ is needed. The super().__init__ is called to pass the string to the base class and the base class has this method implemented, right? I'm not completely sure.

Reply:
The suggested change has been applied. The reason for this check is because some time ago an `Exception` object was passed and this was causing some unexpected errors